### PR TITLE
Massdns and Oneforall using universal wrapper

### PR DESCRIPTION
**Pull request by hiddengearz**

### CODE REVIEW

**Comment by gorums** [comment]
Testing!!!

File: Massdns/readme.md [hunk 1]
```diff
@@ -33,7 +36,8 @@ RUN echo 'export GOPATH=$HOME/go'	>> ~/.profile
 RUN echo 'export PATH=$GOPATH/bin:$GOROOT/bin:$PATH' >> ~/.profile
 RUN . ~/.profile
 RUN git clone https://github.com/blechschmidt/massdns.git && cd massdns && make
-RUN cd /app && wget https://raw.githubusercontent.com/reconness/reconness-agents/master/Massdns/MassdnsWrapper.go
+RUN cd /app && /usr/local/go/bin/go get -u github.com/hiddengearz/reconness-universal-wrapper
+```
```
Comment (gorums) [inline Massdns/readme.md:40]: We need to add the CMD to install the wrapper
RUN cd /root/go/bin/ && ./reconness-universal-wrapper setup -u <reconness username> -p <reconness password> -s <reconness.mydomain.com>

File: OneForAll/readme.md [hunk 2]
```diff
@@ -33,10 +33,10 @@ RUN echo 'export GOROOT=/usr/local/go' >> ~/.profile
 RUN echo 'export GOPATH=$HOME/go'	>> ~/.profile
 RUN echo 'export PATH=$GOPATH/bin:$GOROOT/bin:$PATH' >> ~/.profile
 RUN . ~/.profile
-RUN wget https://raw.githubusercontent.com/reconness/reconness-agents/master/OneForAll/OneForAllWrapper.go
 RUN git clone https://github.com/shmilylty/OneForAll.git
 RUN python3 -m pip install -U pip setuptools wheel
 RUN pip3 install -r /app/OneForAll/requirements.txt
+RUN cd /app && /usr/local/go/bin/go get -u github.com/hiddengearz/reconness-universal-wrapper
```
Comment (gorums) [inline OneForAll/readme.md:40]: We need to add the CMD to install the wrapper
RUN cd /root/go/bin/ && ./reconness-universal-wrapper setup -u <reconness username> -p <reconness password> -s <reconness.mydomain.com>

Comment (gorums) [inline OneForAll/readme.md]: We need to add the CMD to install the wrapper
RUN cd /root/go/bin/ && ./reconness-universal-wrapper setup -u <reconness username> -p <reconness password> -s <reconness.mydomain.com>

File: Massdns/readme.md (hunk 1 above)
Comment (gorums) [inline Massdns/readme.md:38]: We need to add the CMD to install the wrapper
RUN cd /root/go/bin/ && ./reconness-universal-wrapper setup -u <reconness username> -p <reconness password> -s <reconness.mydomain.com>

File: OneForAll/readme.md (hunk 2 above)
Comment (hiddengearz) [reply OneForAll/readme.md]: Fixed

File: Massdns/readme.md (hunk 1 above)
Comment (hiddengearz) [reply Massdns/readme.md:38]: Fixed

**Pull request merged**